include/ttmlir/Dialect/TT/IR/TTOpsTypes.td

```diff
@@ -348,6 +349,9 @@ def TT_DeviceAttr : TT_Attr<"Device", "device", []> {
           llvm_unreachable("Unsupported memory space");
         }
       }
+
+      uint64_t getLayoutSizeBytes(ArrayRef<int64_t> shape, LayoutAttr layout, MemorySpace memorySpace) const;
```

Comment:
Which shape? Lots of shapes going around. :)

Reply:
Updated review, tensorShape